src/Pages/WishlistPage/WishlistPage.js

```diff
@@ -0,0 +1,49 @@
+import React, { useEffect } from 'react'
+import { useNavigate } from 'react-router-dom'
+import './WishlistPage.css'
+import WishlistCard from './WishlistComponents/WishlistCard'
+import Navbar from '../../Components/Navbar/Navbar'
+import { useWishlist } from '../../Contexts/index'
+const axios = require('axios');
+
```

Comment:
changes: use " import axios from 'axios' " instead of require('axios')

Reply:
done